contrib/epee/include/misc_language.h

```diff
@@ -74,10 +74,10 @@ namespace misc_utils
 
   struct call_befor_die_base
   {
-    virtual ~call_befor_die_base(){}
```

Comment:
I didn't even know `=default` could be combined with `virtual`. I don't see any advantages (or disadvantages) - it should be identical in behavior?

Reply:
In addition to being more readable, it would prevent a subtle bug: 

https://stackoverflow.com/a/827205

Reply:
Thanks for the explanation.

Keep in mind I didn't say it does have a bug. I said it can cause a very subtle bug in very very rare cases, because of differences in behaviours [1]. And that link while having some incorrect information, was just to show there are very subtle differences between the two approaches.

1. https://godbolt.org/z/x7odMqb37